func-consumption/package.json

```diff
@@ -38,10 +38,7 @@
     "gulp-typescript": "^5.0.1",
     "gulp-yarn": "^3.0.0",
     "standard": "^17.0.0",
```

Comment:
Why are we keeping standard if we are moving to eslint? `func-throttled` also includes `standard`, which I think would be nice if we could get rid of it too.